proto/config/netcmn.proto

```diff
@@ -83,12 +83,28 @@ message ipspec {
   string subnet = 3;
   string gateway = 5;
   string domain = 6;
+  // ntp and more_ntp are used to specify several NTP servers
```

Comment:
Presumably with ntp being the first and more_ntp being the rest aka the concatenation of ntp+more_ntp specifies all the NTP servers.

Reply:
Hmm, I would not call it "all NTP servers" because there are possibly also NTP servers configured via DHCP.

Reply:
Correct. But it would be helpful to have a comment that it uses the union of ntp and more_ntp as the set of statically/manually configured NTP servers. And it wouldn't hurt to point out that this is due to originally only having a field for a single server.